pre-commit

```diff
@@ -25,9 +25,33 @@ if [[ "$1" == "install" ]]; then
     exit
 fi
 
+save_merge_files() {
+    # MERGE_[HEAD|MODE|MSG]
+    for f in "$root"/.git/MERGE_*; do
+	if [[ -e "$f" ]]; then
+            t=$(basename $f)
+            cp -f "$f" "$t"
+	fi
+    done
+}
+
+restore_merge_files() {
+    for f in MERGE_*; do
+	if [[ -e "$f" ]]; then
```

Comment:
You should need this if statement: the for loop guarantees that a file exists here.

Reply:
I thought so too, but `mv` actually fails if there's no match. "$f" is filed with "MERGE_*" in that case.

`mv: rename /Users/taiki/Source/raw-ipa/.git/MERGE_* to MERGE_*: No such file or directory`

Reply:
Ah, right.  I'm use to `make`, where `$(wildcard MERGE_*)` would be empty if there is no match.

You could use `$(ls MERGE_*)`, I guess.